proto/clients/authenticator.gen.ts

```diff
@@ -742,6 +798,10 @@ const webrpcErrorByCode: { [code: number]: any } = {
   [1000]: UnauthorizedError,
   [1001]: TenantNotFoundError,
   [2000]: EmailAlreadyInUseError,
+  [2001]: AccountAlreadyLinkedError,
```

Comment:
Please see: https://docs.google.com/spreadsheets/d/1wwSciHVQEXzg1JDLhk0V5y3OFPclQ5cYDNLwN6ykRw8/edit?gid=0#gid=0

This is an attempt by Jose to standardize WebRPC error codes. At least we should make sure that codes do not collide or we do not have the same errors with different codes.